src/main/java/com/nha/abdm/wrapper/hiu/hrp/consent/requests/callback/OnStatusRequest.java

```diff
@@ -0,0 +1,20 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.hiu.hrp.consent.requests.callback;
+
+import com.nha.abdm.wrapper.common.responses.ErrorResponse;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class OnStatusRequest {
```

Comment:
nit: Just a suggestion, but can be ignored if not important. Can we have the names of the classes something similar to what is publish in ABDM swagger APIs, so that its easier to relate. In this case, the name was `HIUConsentRequestStatus` in ABDM.

Reply:
thats a good suggestion, we have folder structure in place to organize this. but anyways we can do this cleanup at some point.